### ISSUE TITLE
All secondary pages aren't loading the base css

### ISSUE THREAD

**Issue by pastorryanhayden** [bencentra/centrarium#16]
I'm going to echo what others are saying.  All of the secondary pages (everything but home) isn't loading the css.
I fixed that by changing the css link in includes > head to a static link to "http://myurl.com/css/main.css" but now whenever I click on a link (from any page other than home) it just appends that page name to the current URL.  It seems to be that the {{"URL" | prepend }} tags aren't working (but I'm totally new to jekyll so IDK)
My site is live if you want to check it out http://pastorryanhayden.com

**Comment by pastorryanhayden** [comment]
I've been able to fix a lot of it by changing instances of site.baseurl to just site.url

**Comment by pastorryanhayden** [comment]
I've got everything working.  What I did was change every instance of site.baseurl to just site.url.  I then went and found all of the places where you used a {{page.url | prepend}} tag and just made it two simple tags {{site.url}}/{{page.url}}.  (Atom's find in project is a beutiful thing.)
Anyway.  Thanks for the great theme.  I love bourbon, bitters and neat and it's a great starting point.
